### DISCUSSION TITLE
Quarto 1.5 – Quarto

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11422] [Blog]
# Quarto 1.5 – Quarto

Quarto 1.5 improves Typst support, has some website enhancements like draft handling and announcement bars, adds a native Julia engine, and adds a couple of shortcodes for generating placeholder content.

https://quarto.org/docs/blog/posts/2024-07-11-1.5-release/

**Comment by aborruso** [comment]
thank you very much!!
